packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -18,24 +25,60 @@ class AndroidCameraCameraX extends CameraPlatform {
     CameraPlatform.instance = AndroidCameraCameraX();
   }
 
-  /// ProcessCameraProvider used to get list of cameras. Visible only for testing.
+  // Instances used to access camera functionality:
+
+  /// The [ProcessCameraProvider] instance used to access camera functionality.
   @visibleForTesting
   ProcessCameraProvider? processCameraProvider;
 
-  /// Camera selector used to determine which CameraInfos are back cameras.
+  /// The [Camera] instance returned by the [processCameraProvider] when a [UseCase] is
+  /// bound to the lifecycle of the camera it manages.
+  @visibleForTesting
+  Camera? camera;
```

Comment:
nit: It may be worth moving all the `@visibleForTesting` fields and methods to a separate class and pass that class as an optional parameter in a constructor. Calling it something like `AndroidCameraXProxy`. This could help organize the code a bit. You can decide which is easier to use.

Reply:
I would honestly find this kind of confusing, especially since this would involve some of the major use case instances. Are you open to reconsidering this later on? I could see it helping organizationally if there ends up being a lot of different fields (especially booleans and things that aren't necessarily CameraX objects) that need to be visible or there are a bunch more methods we'll need to create to mock.